Backend/dev/integration-tests/collections/AssociationHistoryFlow/01-AssociationHistoryFix.json

```
@@ -0,0 +1,758 @@
+              "    pm.expect(activeAssocs).to.have.length(0,",
+              "        'Expected no active fleet_driver_association but found: ' + JSON.stringify(activeAssocs));",
+              "    console.log('PASS: fleet_driver_association is_active=false');",
+              "});"
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "name": "--- Scenario 2: Active RC Association Cleanup ---",
+      "request": { "method": "GET", "url": { "raw": "about:blank", "raw": "about:blank" }, "body": { "mode": "raw", "raw": "" } },
```

Biome (2.5.3)

[error] The key raw was already declared.

(lint/suspicious/noDuplicateObjectKeys)